third_party/amd/lib/TritonAMDGPUTransforms/OptimizeEpilogue.cpp

```diff
@@ -68,6 +95,21 @@ class BypassEpilogueSMEM : public mlir::RewritePattern {
         !valType.getEncoding().isa<triton::gpu::BlockedEncodingAttr>())
       return mlir::failure();
 
+#ifdef USE_ROCM
```

Comment:
do you still need the #ifdef? If not can you remove them

Reply:
yeah, the AMD backend still needs ifdef at this time

Reply:
This is an AMD specific file so no `ifdef` is necessary to build.
But this file is still very similar to Nvidia one, so I would like to keep this guards to highlight differences.

Reply:
ok interesting. The Nvidia pass seems to be unused so I think we should delete it. @ptillet is there any reason for us to keep it?